### PR TITLE
perf: Only show builds on first group_overview page

### PR DESCRIPTION
**Pull request by perlpunk**
This is a simple first mitigation to avoid crawlers going through all comment pages to always load the builds again.

Issues:
* https://progress.opensuse.org/issues/204954
* https://progress.opensuse.org/issues/204963

Will probably be obsoleted when comments are loaded via ajax.

group-overview-comments-no-builds

### CODE REVIEW

**Comment by perlpunk** [comment]
The container check is now broken for unknown reasons, hopefully only temporarily: https://github.com/os-autoinst/openQA/actions/runs/30553320055/job/90908315618


Edit: Ah, now I get it. It was also having problems with loading the container at all when I retriggered, so I thought it was a sporadic problem.
```
Writing manifest to image destination
container/single-instance/Dockerfile:30 DL3025 warning: Use arguments JSON notation for CMD and ENTRYPOINT arguments
make: *** [Makefile:474: test-check-containers] Error 1
```
Will create a PR

#7694

**Review by Martchus — APPROVED** [review]

**Comment by perlpunk** [comment]
Alternatively we could still show the builds for logged in users.

**Comment by Martchus** [comment]
I think this is good as-is. If I'm logged in I'd also like to benefit from faster loading times when going through pages of comment and considering the builds are still shown on the first page there's no loss. (Not that I ever do that.)

**Comment by okurz** [comment]
> Alternatively we could still show the builds for logged in users.

That's also a nice idea but maybe a bit more evolved to get done properly. Your first approach sounds good enough for now

**Comment by perlpunk** [comment]
I changed it to also skip fetching tags because without builds we don't need them.

**Comment by perlpunk** [comment]
> > Alternatively we could still show the builds for logged in users.
> 
> That's also a nice idea but maybe a bit more evolved to get done properly. Your first approach sounds good enough for now

I would just check `$self->current_user`, so that looks easy to me. But we probably don't need it.

**Review by okurz — APPROVED** [review]
